src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,18 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
+        Machine track = new Track();
 
+        Machine[] machines = new Machine[]{excavator, bulldozer, track};
+        for (Machine machine : machines) {
+            machine.doWork();
+        }
+
+        for (Machine machine1 : machines) {
```

Comment:
```suggestion
        for (Machine machine : machines) {
```
i don't see any reasons to change naming in this loop

Reply:
yep, exactly, i didn't notice it, done